### PR TITLE
Fix/shipping estimate.especification

### PR DESCRIPTION
**Pull request by matheusMFCosta**
#### What is the purpose of this pull request?
fix the incorrect value shown on the specification filed of the deliveryTime

#### What problem is this solving?
-  translate and place the correct specification of the deliveryTime on `ShippingSimulator`
-  remove unnecessary code of the of locale builder

#### How should this be manually tested?
run the project at the product detail of the dreamStore

#### Screenshots or example usage
screen shot 2019-01-09 at 09 40 16

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: messages/en-US.json [hunk 1]
```diff
@@ -28,7 +28,11 @@
   "pricing.interest-free": "interest-free",
   "loading": "Loading…",
   "shipping.label": "Calculate shipping",
-  "shipping.eta": "up to {eta} days",
+  "shipping.eta": "up to {eta} {especification}",
```
Comment (lucasecdb) [inline messages/en-US.json]: shouldn't it be `specification` instead of `especification`?

Comment (matheusMFCosta) [reply messages/en-US.json]: heheheh sim

File: react/components/ShippingSimulator/components/ShippingTable.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export default class ShippingTable extends Component {
       })),
     }),
   }
-
+
```
Comment (lucasecdb) [inline react/components/ShippingSimulator/components/ShippingTable.js]: remove? :pick:

File: react/components/ShippingSimulator/components/ShippingTableRow.js [hunk 3]
```diff
@@ -3,6 +3,21 @@ import PropTypes from 'prop-types'
 import { intlShape, injectIntl } from 'react-intl'
 import classNames from 'classnames'
 
+const datesMap = {
+  bd: "shipping.busniess-days",
```
Comment (lucasecdb) [inline react/components/ShippingSimulator/components/ShippingTableRow.js]: use single quotes :pick:

File: react/components/ShippingSimulator/components/ShippingTableRow.js [hunk 4]
```diff
@@ -3,6 +3,21 @@ import PropTypes from 'prop-types'
...
+  d: "shipping.calendar-days",
+  m: "shipping.minutes",
+  h: "shipping.hours",
+  default: "shipping.busniess-days"
+}
+
+const deliveryTimeParser = (deliveryTime) => {
+  const deliveryTimeRegex = /^(\d+)([a-z]+)$/i
+  const eta = deliveryTimeRegex.exec(deliveryTime) 
+  if(!eta) return { value: "", key: "" }
```
Comment (lucasecdb) [inline react/components/ShippingSimulator/components/ShippingTableRow.js]: single quotes :pick:

File: react/components/ShippingSimulator/components/ShippingTableRow.js [hunk 5]
```diff
@@ -12,13 +27,11 @@ const ShippingTableRow = ({ name, shippingEstimate, price, intl }) => {
...
 
-  let etaText, valueText
-
-  if (shippingEstimate === undefined) {
-    etaText = '-'
-  } else {
-    etaText = intl.formatMessage({ id: 'shipping.eta' }, { eta: shippingEstimate })
-  }
+  const deliveryTime = deliveryTimeParser(shippingEstimate)
+  const deliveryTimeEspecificationText = intl.formatMessage({id:deliveryTime.specification})
```
Comment (lucasecdb) [inline react/components/ShippingSimulator/components/ShippingTableRow.js]: add spaces inside the curly brackets :pick:

File: react/components/ShippingSimulator/components/ShippingTableRow.js [hunk 6]
```diff
@@ -12,13 +27,11 @@ const ShippingTableRow = ({ name, shippingEstimate, price, intl }) => {
...
+  const deliveryTimeText = intl.formatMessage({ id: 'shipping.eta' }, { eta: deliveryTime.value, specification: deliveryTimeEspecificationText})
```
Comment (lucasecdb) [inline react/components/ShippingSimulator/components/ShippingTableRow.js]: missing space before the closing bracket :pick:

File: react/components/ShippingSimulator/components/ShippingTableRow.js [hunk 7]
```diff
@@ -12,13 +27,11 @@ const ShippingTableRow = ({ name, shippingEstimate, price, intl }) => {
...
+  
+  let  valueText
```
Comment (lucasecdb) [inline react/components/ShippingSimulator/components/ShippingTableRow.js]: extra space :pick:

**Pull request closed without merging**